sqlbucket/cli.py

```diff
@@ -51,17 +61,31 @@ def run_job(sqlbucket, name, db, fstep, tstep, to_date, from_date,
         logger.info('Variables used')
         logger.info(submitted_variables)
 
-        etl = sqlbucket.load_project(
-            project_name=name,
-            connection_name=db,
-            variables=submitted_variables
-        )
-        if rendering:
-            etl.render(from_step=fstep, to_step=tstep, group=group)
+        # included dbs
+        if db:
+            dbs = db.split(',')
+        elif all:
+            dbs = every_dbs(sqlbucket.env_name)
```

Comment:
Based on my previous comment, I think you could do it this way:

```python
if deb:
    dbs =db.split(',')
elif all:
    dbs = sqlbucket.connections.keys()
```

not entirely sure of the code here, but either way something along those lines should work.

Reply:
Done